pkgs/top-level/all-packages.nix

```diff
@@ -9518,15 +9518,22 @@ in
 
   rust_1_44 = callPackage ../development/compilers/rust/1_44.nix {
     inherit (darwin.apple_sdk.frameworks) CoreFoundation Security;
+    llvmPackages = llvmPackages_9;
   };
   rust_1_45 = callPackage ../development/compilers/rust/1_45.nix {
```

Comment:
It might be nice to leave a comment explaining why the older versions are left available.

Reply:
Done